index.js

```diff
@@ -45,7 +45,7 @@ xapp.init(
   () => (config.GRAVITY_XAPP_TOKEN = xapp.token)
 )
 
-app.get("/favicon.ico", (req, res) => {
+app.get("/favicon.ico", (_req, res) => {
```

Comment:
the files with `// @ts-check` get a warning on these unused parts, so I just use the underscore notation to make it obvious (and to kill the warring ) as I see them,

Reply:
👍